src/framework/preferences/messages/linksControls.json

```diff
@@ -1,3 +1,4 @@
 {
-    "linksControlsLabel": "Links & buttons"
+    "emphasisDescr": "Emphasize input options",
```

Comment:
I think I need to walk through this with you @maozillah and/or @jobara to understand what all the changes are.